module/multimedia/src/Infrastructure/Service/SuffixGeneratingServiceInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Multimedia\Infrastructure\Service;
+
+interface SuffixGeneratingServiceInterface
```

Comment:
Is the infrastructure layer an appropriate one? I don't think so tbh, more like an application, or even a domain one.